src/jsdoc.ts

```diff
@@ -270,7 +264,12 @@ export function toString(tags: Tag[], escapeExtraTags: string[] = []): string {
 
   let out = '';
   out += '/**\n';
+  const emitted = new Set<string>();
   for (let tag of tags) {
+    if (emitted.has(tag.tagName) && SINGLETON_TAGS.has(tag.tagName)) {
```

Comment:
This vaguely feels like it might belong in the merge() function, which handles multiple copies of other tags as well.  Another way of saying this is that this toString function is careful to emit exactly what it was given, rather than trying to fix up its input, so the fixing maybe belongs at a different layer (not necessarily merge). 

Maybe for now split this into a cleanup() function that toString() always calls, and cleanup() does the singleton-filtering?  Or maybe I am worrying too much about purity here and what you have is fine.  Your call.